packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx

```diff
@@ -60,6 +60,7 @@ const OrganizationProfileSection = () => {
 const OrganizationDangerSection = () => {
   const { organization, membership } = useCoreOrganization();
   const { navigate } = useRouter();
+  const adminDeleteEnabled = organization?.adminDeleteEnabled;
```

Comment:
🙃 Could move this after the check on lines 65-67 and get rid of the optional `?`